identity_provider_test.go

```diff
@@ -227,7 +227,7 @@ func TestIDPHTTPCanHandleSSORequest(t *testing.T) {
 		a, _ := url.QueryUnescape(validRequest)
 		b, _ := base64.StdEncoding.DecodeString(a)
 		c, _ := ioutil.ReadAll(flate.NewReader(bytes.NewReader(b)))
-		d := bytes.Replace(c, []byte("<AuthnRequest"), []byte("<AuthnRequest ::foo=\"bar\""), 1)
+		d := bytes.Replace(c, []byte("<AuthnRequest"), []byte("<AuthnRequest ::foo=\"bar\">]]"), 1)
```

Comment:
Interesting change. Is this extra closing ">]]" part of the new version?

Reply:
No.. these tests have been copied from older version of crewjam/saml and they were already failing (which we have been ignoring)